### PR TITLE
Tweak search results with SortBy algolia widget

### PR DESCRIPTION
**Pull request by DrShpongle**
This is a draft PR related to the https://github.com/eggheadio/egghead-next/issues/805

This is how it looks:
Resources for Web Developers in 2021 egghead io 2021-08-19 at 2 57 58 PM

Also fixed some weird spacings (paddings and negative margins) on general search layout and minor changes on CSS curated page (will update other curated pages if those changes accepted).

What about `SortBy` algolia widget we need to create related indices on Algolia side. Is there a way to have a test Algolia instance to play with in dev?

### CODE REVIEW

File: src/components/search/index.tsx [hunk 1]
```diff
@@ -228,25 +174,111 @@ const Search: FunctionComponent<SearchProps> = ({
...
+                    <CuratedTopicPage topic={topic} />
+                  )}
+              </div>
+            )}
+
+            <div className="dark:bg-gray-900 bg-gray-50 mt-6">
+              <ScrollTo scrollOn="page" />
+
+              {/* SortBy widget goes below. Related indicies should be added on Algolia side before */}
+              {/* <div className="flex justify-between items-end pb-4 mb-6 border-b border-gray-200">
```
Comment (DrShpongle) [inline src/components/search/index.tsx:219]: commented out for now

**Comment by laurosilvacom** [comment]
@DrShpongle We don't have applications per environment in Algolia. Would using the different staging indices be helpful?

CleanShot 2021-08-19 at 11 51 40@2x

**Comment by joelhooks** [comment]
we definitely have algolia partitioned by environment. There is content_dev and content_prod

**Comment by MaggieAppleton** [comment]
Anything we can do to move this forward? Are we stuck on the Algolia integration?

**Comment by DrShpongle** [comment]
Yes, I have stuck with Algolia integration. If I understand correct, all `SortBy` widget does is switch the indices. So we have to create the indices for each of criteria (published, duration, popular, rated, updated), but I'm confused with how to rank it by these values and he we can pass for example rating or publish date to algolia.

**Comment by DrShpongle** [comment]
closed in favor of https://github.com/eggheadio/egghead-next/pull/870

**Pull request closed without merging**